bin/live/pycbc_live_combine_single_fits

```diff
@@ -89,14 +116,15 @@ for f in files:
         analysis_day = int(m.group(3))
         n_since_epoch = (datetime.date(analysis_year, analysis_month,
                                        analysis_day)
-                         - datetime.date(2000,1,1)).days
+                         - epoch).days
         analysis_dates += [n_since_epoch]
+
     for ifo in args.ifos:
         if ifo not in fits_f:
+            logging.info(f + " has no " + ifo + " triggers")
```

Comment:
This condition is expected fairly often, so I would not spam the stderr so much.